### PR TITLE
Automated df-structures update for DF 0.44.09

### PR DESCRIPTION
**Pull request by BenLubar**
This is an automated pull request. Only one such pull request should be made for each Dwarf Fortress release. If this bot is malfunctioning, contact BenLubar on IRC.

### CODE REVIEW

**Comment by BenLubar** [comment]
Better diff generated by `git diff --minimal`

https://gist.github.com/BenLubar/7665f0a5a3e2677e760738cc4afc6b10

File: symbols.xml [hunk 1]
```diff
@@ -13361,6 +17396,2046 @@
...
+        <vtable-address name='renderer_offscreen' value='0x2430048'/>
+        <vtable-address name='renderer_once' value='0x24341c8'/>
+        <vtable-address name='renderer_opengl' value='0x2434228'/>
+        <vtable-address name='renderer_partial' value='0x24342e8'/>
+        <vtable-address name='renderer_vbo' value='0x2434348'/>
+        <vtable-address name='resource_allotment_specifier_ammost' value='0x2433a88'/>
+        <vtable-address name='resource_allotment_specifier_anvilst' value='0x2433b00'/>
+        <vtable-address name='resource_allotment_specifier_armor_bodyst' value='0x24338f8'/>
+        <vtable-address name='resource_allotment_specifier_armor_bootsst' value='0x2433970'/>
+        <vtable-address name='resource_allotment_spe
```
Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

Comment (BenLubar) [inline symbols.xml]: conflict

**Comment by BenLubar** [comment]
My review seems to be rendering wrong, so here's the conflicts in a nice text file:

https://gist.github.com/BenLubar/302fb5008d5f0b9dc4e28695767e93a3

**Comment by lethosor** [comment]
Renderer is the only important one, for rendermax, and I believe it's usually the third. It's easy enough to verify at runtime, and possibly statically too. The others are generally unimportant and might be pure virtual anyway.

**Pull request merged**